client/src/containers/itemView.js

```diff
@@ -65,15 +69,23 @@ class ItemViewContainer extends Component {
   }
   
   render() {
-    const { ancestors, item, children, location } = this.props;
+    const { ancestors, item, children, me, location } = this.props;
+    const NODES = getNodes();
     
     if (item.type !== ROOT_NODE && isNil(item.fields)) {
       return <NotFoundPage />;
     }
 
+    let canEdit = false;
+    if (!isNil(me)) {
+      if (item.type === ROOT_NODE || item.fields.owner === me['_id']) {
+        canEdit = true;
+      }
+    }
```

Comment:
`owner` is not necessary the only one who can edit. For example group membership could give write access.